dex/src/main/scala/com/wavesplatform/dex/api/websockets/WsOrder.scala

```diff
@@ -62,6 +52,12 @@ object WsOrder {
     )
   }
 
+  def fromDomain(ao: AcceptedOrder, status: OrderStatus, fa: Double, ff: Double, awp: Double)(implicit efc: ErrorFormatterContext): WsOrder =
+    (ao.isMarket match {
+      case true => fromDomain(ao, status)
+      case _ =>  WsOrder(ao.id, status = status.name.some)
+    }).copy(ao.id, filledAmount = fa.some, filledFee = ff.some, avgWeighedPrice = awp.some)
+
```

Comment:
1. You don't need `ErrorFormatterContext` here I guess
2. Please don't use pattern matching for booleans, it's better to use simple conditional logic:

```suggestion
  def fromDomain(ao: AcceptedOrder, status: OrderStatus, fa: Double, ff: Double, awp: Double): WsOrder = {
    (if (ao.isMarket) fromDomain(ao, status) else WsOrder(ao.id, status = status.name.some)).copy(ao.id, filledAmount = fa.some, filledFee = ff.some, avgWeighedPrice = awp.some)
    }
```